edgekv/lib/README.md

```diff
@@ -17,12 +17,13 @@ The edgekv.js file must be included alongside `edgekv_tokens.js` which includes
 	/**
 	 * Constructor to allow setting default namespace and group
 	 * These defaults can be overriden when making individual GET, PUT, and DELETE operations
-	 * @param {string} [namespace] the default namespace to use for all GET, PUT, and DELETE operations
+	 * @param {string} [$0.namepsace] the default namespace to use for all GET, PUT, and DELETE operations
 	 * 		Namespace must be 32 characters or less, consisting of A-Z a-z 0-9 _ or -
-	 * @param {string} [group] the default group to use for all GET, PUT, and DELETE operations
+	 * @param {string} [$0.group] the default group to use for all GET, PUT, and DELETE operations
 	 * 		Group must be 128 characters or less, consisting of A-Z a-z 0-9 _ or -
+	 * @param {number} [$0.num_retries_on_timeout=0] the number of times to retry a GET requests when the sub request times out
 	 */
-	new EdgeKV(namespace = "default", group = "default")
+	new EdgeKV({namespace = "default", group = "default", num_retries_on_timeout = 0})
```

Comment:
Is there a reason this defaults to 0 while others default to null? If not, let's make them consistent.

Reply:
yeah, 0 is what the class will use if the other values are null.
i.e., if you don't provide a value for the method, it will default to the class value of 0